lib/issue_exporter/version.rb

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2015 Scott Williams
 
 module IssueExporting
-  VERSION = '0.1.0'
+  VERSION = '0.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.